pkg/registry/image_factory.go

```diff
@@ -3,19 +3,16 @@ package registry
 import (
 	"time"
 
-	"github.com/buildpack/imgutil"
-	"github.com/buildpack/imgutil/remote"
 	"github.com/google/go-containerregistry/pkg/authn"
-	"github.com/pkg/errors"
 )
 
 type ImageFactory struct {
 	KeychainFactory KeychainFactory
 }
 
 func (f *ImageFactory) NewRemote(imageRef ImageRef) (RemoteImage, error) {
```

Comment:
As an extension of our previous comment, can we update this as well to say `Get/FetchRemote` because it will never be our intent on actually creating a new remote image